tools/esdlcmd/esdlcmd_manifest.cpp

```diff
@@ -0,0 +1,1409 @@
+         *
+         * @param xpp pointer to the pull parser
+         * @param root reference to the 'Transform' node read
+         * @return int type of the last node read
+         */
+        int processTransform(IXmlPullParser* xpp, StartTag& root)
+        {
+            int type;
+            StartTag stag;
+            EndTag etag;
+            // Track the node level nesting of nodes handled in this function only
+            int level = 1;
+
+            // To preserve order, the expected format for transforms is:
+            // <Transform>
+            //     <![CDATA[...XSLT...]]>
+            // </Transform>
+            appendStartTag(xpp, root, false, true);
+            appendStartCDATA();
+
+            do
+            {
+                type = xpp->next();
+                switch (type)
+                {
+                    case XmlPullParser::START_TAG:
+                        xpp->readStartTag(stag);
+                        if (isManifestNodeNamed(stag, MANIFEST_TAG_INCLUDE))
+                            processInclude(xpp, stag, INCLUDE_XSLT, nullptr);
+                        else
+                        {
+                            appendStartTag(xpp, stag);
+                            level++;
+                        }
+                        break;
+
+                    case XmlPullParser::CONTENT:
+                        appendNonWhitespaceContent(xpp);
+                        break;
+
+                    case XmlPullParser::END_TAG:
+                        xpp->readEndTag(etag);
+
+                        // We're about to encounter the root <Transform> node, so close
+                        // out the CDATA section we injected first
+                        if (isRootEndTagNamed(etag, "Transform", level))
+                        {
+                            appendEndCDATA();
+                            appendEndTag(etag, false);
+                        }
+                        else
+                            appendEndTag(etag);
+
+                        level--;
+                        break;
+
+                    case XmlPullParser::END_DOCUMENT:
+                        throw makeStringExceptionV(ESDL_MANIFEST_ERR, "Unexpected end of document at line #%d, column #%d", xpp->getLineNumber(), xpp->getColumnNumber());
+                        break;
+                }
+                lastType = type;
+            }
+            while (level > 0);
+
+            return type;
+        }
+
+        /**
+         * @brief Process the 'ServiceDefinition' node and children.
+         *
+         * The 'ServiceDefinition' node is renamed on output to 'Definition'. Its contents are enclosed
+         * with CDATA.
+         *
+         * @param xpp pointer to the pull parser
+         * @param root reference to the 'ServiceDefinition' node read
+         * @return int type of the last node read
+         */
+        int processServiceDefinition(IXmlPullParser* xpp, StartTag& root)
```

Comment:
Can this be renamed to something like `processEsdlServiceDefinition`?

Reply:
sure- or perhaps EsdlDefinition? Either way if I change it should I change the element name to match?

Reply:
If a bundle is viewed as a superset of a binding, maybe `processBundleDefinitions` versus `processBindingDefinition`. I wouldn't complain about consistency between the markup and function name.

Reply:
Tony said he'd prefer EsdlDefinition, so I'll change that name for the function and for the manifest namespace element.